ha/setup/ha_setup.py

```diff
@@ -855,6 +855,16 @@ def main(argv: list):
             log_path = Conf.get(const.HA_GLOBAL_INDEX, f"LOG{_DELIM}path")
             log_level = Conf.get(const.HA_GLOBAL_INDEX, f"LOG{_DELIM}level")
             Log.init(service_name='ha_setup', log_path=log_path, level=log_level)
+        elif sys.argv[1] == "cleanup":
+            if not os.path.exists('const.HA_CONFIG_FILE'):
+                a_str = f'Cleanup can not be proceed as \
+                           HA config file: {const.HA_CONFIG_FILE} \
+                           is missing. Either cleanup is already done or there \
+                           is some other problem'
+                Log.warn(f'{a_str}')
+                sys.stderr.write(a_str)
+                return 1
```

Comment:
```
sys.stderr.write(a_str)
```
change to 
```
sys.stdout.write(f"Warning: {a_str})
```
It is warning so return code need to 0

Reply:
Done

Reply:
Removing the part to log the warning. Hence, now keeping return code as 1.

Reply:
Instead of the hard-coded '1' here, can we use some standard error code from errno?

Reply:
I think it should be 0

as it is warning not error.
Also if we are returning error then this bug still exists
```
Cleanup command fails if run multiple time
```

Reply:
Ok. Returning 0 now

Reply:
@ajay-paratmandali  Lets resolve this conversation if ok.